pkg/components/networking/ingressnodefirewall/component.go

```diff
@@ -14,7 +14,9 @@ var IngressNodeFirewallComponent = Component{
 		Name:                 "Networking / ingress-node-firewall",
 		Operators:            []string{},
 		DefaultJiraComponent: "Networking / ingress-node-firewall",
-		Matchers:             []config.ComponentMatcher{},
+		Matchers: []config.ComponentMatcher{
+			{Suite: "Egress-ingress related networking scenarios"},
```

Comment:
`Egress-ingress related networking scenarios` belong to openshiftsdn and ovnkubernetes component

Here should be Suite: "[sig-networking] SDN infw"

Reply:
```suggestion
			{Suite: "SDN infw"},
```